cypress/support/cypress-support/src/utils.js

```diff
@@ -687,7 +687,10 @@ function checkForSecondaryAppId(appId) {
     if (appId === CONSTANTS.SECONDARY_THIRD_PARTY_APP) {
       envAppIdKey = CONSTANTS.SECONDARY_THIRD_PARTY_APP_ID;
       return getEnvVariable(CONSTANTS.SECONDARY_THIRD_PARTY_APP_ID);
-    } else if (getEnvVariable(appId) !== null && getEnvVariable(appId) !== undefined) {
+    } else if (
+      getEnvVariable(appId, false) !== null &&
+      getEnvVariable(appId, false) !== undefined
+    ) {
```

Comment:
need to mention in readme about https://github.com/comcast-firebolt/xumo-remote-config-module/pull/101/files#diff-6663f66a544d92b19aa9ce88d1238b34917b5c6bbd205ef6e1de0e958857e5caR53 log to be included in config module to show the report with screenshot hyperlink